tests/selenium_test_websocket.py

```diff
@@ -0,0 +1,66 @@
+from selenium import webdriver
+
+def log_in(driver):
+    driver.get('http://dashboard.openwisp.org/admin/login/')
+    if not 'admin/login' in driver.current_url:
+        return
+    username_input = driver.find_element_by_name('username')
+    password_input = driver.find_element_by_name('password')
+    username_input.send_keys('admin')
+    password_input.send_keys('admin')
+    log_in_button = driver.find_element_by_class_name('submit-row').find_element_by_xpath('//input[@type="submit"]')
+    log_in_button.click()
+
+def create_location(driver):
```

Comment:
Can you please make a another class to put all the websocket related tests together, just for the sake of keeping stuff together! :smile: